src/PowerModels.jl

```diff
@@ -46,6 +48,7 @@ include("core/data.jl")
 include("core/ref.jl")
 include("core/base.jl")
 include("core/types.jl")
+include("core/helpers.jl")
```

Comment:
So far I have followed a convention that function overloads from IM are placed in the same location as the original functions appear in IM.  I think it makes it easier to find things.

What do you think about moving these into base.jl to be consistent with that convention?

Reply:
Ah, that makes sense for the sake of consistency. Maybe placing them at the bottom of `base.jl`, after the definition of more important functions, is the best way to do this.